app/javascript/components/AddMotorbikeForm.js

```diff
@@ -0,0 +1,164 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import loading from '../assets/images/loading.gif';
+
+const AddMotorbikeForm = ({ onAddMotorbike }) => {
```

Comment:
- [ ] When I try to add a bike, the item is recorded twice:
As you can see, the records are almost identical:

![image](https://github.com/MenendezON/book-an-appointment/assets/57421823/6a467da6-c365-4abd-835c-b8b9f9dd4983)

- [ ]  When I try to add a reservation, the record is created twice:

![image](https://github.com/MenendezON/book-an-appointment/assets/57421823/48dc3135-10d7-4637-8bf7-1b021693668a)

![image](https://github.com/MenendezON/book-an-appointment/assets/57421823/aaab0fc0-5b0e-4b55-ba52-c3c5bd9fc03a)

Kindly, make sure your records are only created one time when it comes to adding and reserving bikes. 👍